web/src/main/webapp/hillview.css

```diff
@@ -107,6 +107,7 @@ table {
     background: linear-gradient(#f9f9f9,#e4e4e4);
     border: 1px solid black;
     font-weight: bold;
+    padding: 2px 5px;
```

Comment:
If you want to change the style you will have to justify these changes.

Reply:
The padding is intended to separate the label from the column border. 
![image](https://user-images.githubusercontent.com/7077197/88795651-03de6f80-d166-11ea-9581-3276842cbd1a.png)
Should I undo this?

Reply:
I undo this. There is a side effect that cause the first column to expand in width every time I execute `show`

Reply:
There may be an inner box where you can apply this.

Reply:
This is not significant for this PR, I am using the original css. Previously, I added the padding because the label is too close to the column border